datahub-web-react/src/app/homeV3/modules/hierarchyViewModule/treeView/DepthMargin.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import styled from 'styled-components';
+
+const DepthMarginContainer = styled.div<{ $depth: number }>`
+    margin-left: calc(8px * ${(props) => props.$depth});
+`;
+
+interface Props {
+    depth: number;
+}
+
+export default function DepthMargin({ depth }: Props) {
+    return <DepthMarginContainer $depth={depth} />;
+}
```

Comment:
yeah it looks like this is getting the right depth for the comment i made earlier about children not being at the right depth. so i'm not sure exactly what's going on here